react-app/src/app/screens/Game/components/Info/layout.js

```diff
@@ -12,3 +15,12 @@ function Info({ status, movesList }) {
 }
 
 export default Info;
+
+Info.propTypes = {
+  status: PropTypes.string,
+  movesList: PropTypes.element.isRequired
```

Comment:
element?

Reply:
I wasn't sure about this one, because movesList is an Array of Button components (JSX Elements). That's why I used PropTypes.element. I just changed it to:

`movesList: PropTypes.arrayOf(PropTypes.element).isRequired`